src/SFA.DAS.RoatpGateway.Web/Validators/RoatpGatewayApplicationViewModelValidator.cs

```diff
@@ -85,8 +99,21 @@ public async Task<ValidationResponse> Validate(RoatpGatewayApplicationViewModel
                             validationResponse.Errors.Add(new ValidationErrorDetail("OptionRejectedText", TooManyWords));
                         }
                     }
+
+                    if (string.IsNullOrEmpty(viewModel.OptionExternalRejectedText))
+                    {
+                        validationResponse.Errors.Add(new ValidationErrorDetail("OptionExternalRejectedText", ErrorEnterExternalComments));
+                    }
+                    else
+                    {
+                        var wordCount = viewModel.OptionExternalRejectedText.Split(new[] { " " }, StringSplitOptions.RemoveEmptyEntries).Length;
+                        if (wordCount > 150)
+                        {
+                            validationResponse.Errors.Add(new ValidationErrorDetail("OptionExternalRejectedText", TooManyWords));
+                        }
```

Comment:
Does this tally up with the view? I think I saw 500 words there

Reply:
Yep.. I think you're right here @martin-lush. All the external comments seem to have a 500 word limit, though this is really for @chrisfoster76 to update.